src/main/drivers/usb_msc_common.c

```diff
@@ -127,6 +129,8 @@ void mscWaitForButton(void)
     delay(DEBOUNCE_TIME_MS);
     while (true) {
         mscTask();
+        // Ensure a service window exists for interrupts
+        asm("NOP");
```

Comment:
Do you want to make it __NOOP;?

Reply:
Done. I've also changed the occurrence in `rx_sx1280.c`.